packages/datagrid/src/components/DefaultCellRenderer/DefaultCellQualityIndicator.component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as RuleCore from '@talend/rule-core';
+
+const { QualityDetailsIndicator } = RuleCore.components;
+
+type DefaultCellQualityIndicatorPropTypes = {
+	value: {
+		quality: -1 | 0 | 1;
```

Comment:
did you know about
https://github.com/Talend/ui-ee/blob/fe6ea6aa624e172bc45fbf170a381fa9cfe2afba/DataMapper/src/types/avroSchema.types.ts#L31

do you think we should factorize or reuse one in the other?

Reply:
This is supposed to change with the future Sample API (that will provide details about rules etc), so not sure it's worth the trouble for now